### ISSUE TITLE
Aiming for keyboard-only inputs

### ISSUE THREAD

**Issue by afrey15** [famegames/fire_and_ice#166]
**Context** 
Players using keyboard only inputs cannot aim when shooting fire and ice. They need some way to be able to aim and fire.

**Idea**
Create an aiming tick to control aiming direction. Once a character is holding an item and presses the throw button, the aiming tick appears and they can then press the up and down arrows to adjust the direction of the throw up to straight up or straight down. A little red tick appears during this step that shows the current aiming direction. The default direction should be slightly up from 90 degrees on either side.

**Comment by Regulator13** [comment]
New idea: make the unused up button into an aim button that goes 180 degrees around one side. The little red tick will appear in the direction they are firing and will show constantly

**Comment by afrey15** [comment]
Hrmm
